### PR TITLE
Adds Additional Electronic_Assembly Designs

### PR DESCRIPTION
**Pull request by JStheguy**
:cl: JStheguy
add: Added 10 new assembly designs to the integrated circuit printer, the difference from current designs is purely aesthetics.
imageadd: Added the icons for said new assembly designs to electronic_setups.dmi, changed the current electronic mechanism and electronic machine sprites.
/:cl:

[why]: Lets IC based designs within the same size category not all look exactly the same. Also big grabby robot arms for the million throw-bot designs either complete or in development.

Tired of your mean meme machines all looking the same? Introducing additional casing designs!

ss 2017-12-11 at 03 22 19
Picture of all the new casings, along with the pre-existing casings.

Update: New old electronic mechanism and electronic machine sprites
ss 2017-12-11 at 10 50 43

### CODE REVIEW

**Comment by nnystyxx** [comment]
As a fan of assemblies, yes, please, I want these. The single-sprite ones are boring and also the small one doesn't even look too great so I would love to see this merged in.

**Comment by arsserpentarium** [comment]
I like it.But maybe different assembly designs should have different properties?

**Comment by epochayur** [comment]
wish I could thumbs-up this twice

**Comment by ACCount12** [comment]
Please, resprite the default ones too. They just suck when compared to the new ones.

**Comment by ACCount12** [comment]
The screens on computer-like and laptop-like assembly look really weird.

File: code/modules/integrated_electronics/core/assemblies.dm [hunk 1]
```diff
@@ -403,7 +403,17 @@
 
 	return acting_object.drop_location()
 
+/obj/item/device/electronic_assembly/calc
+	icon_state = "setup_small_calc"
```
Comment (ACCount12) [inline code/modules/integrated_electronics/core/assemblies.dm]: The names are all the same? This would break savefiles.

**Comment by GunHog** [comment]
Beautiful!!

File: code/modules/integrated_electronics/core/assemblies.dm (hunk 1 above)
Comment (JStheguy) [reply code/modules/integrated_electronics/core/assemblies.dm]: Oh, shit, yeah. I'll fix that while I'm giving the default sprites a facelift

**Comment by JStheguy** [comment]
I gave the old electronic mechanism and electronic machine sprites a facelift.
ss 2017-12-11 at 10 50 43

**Comment by Bawhoppen** [comment]
@WJohn For sprite review

**Comment by ACCount12** [comment]
Is that review enough?
default

**Comment by MrMaclang** [comment]
Give the one with hands a built in grabber

**Comment by ACCount12** [comment]
Built in things cause problems. Not sure what kind of problems, but this feature just isn't ready.

**Comment by JStheguy** [comment]
>Built in things cause problems.
Weak!

**Pull request merged**